### PR TITLE
[DEV-1387] Add `column_cleaning_operations` to view object

### PR DESCRIPTION
**Pull request by gee-senbong**
## Description

This PR introduces `column_cleaning_operations` property to view object.

## Type of Change

Label this pull request to place it under the correct category in Release Notes:

|        **Pull Request Label**         | **Category in Release Notes** |
|:-------------------------------------:|:-----------------------------:|
|       `enhancement`, `feature`        |          🚀 Features          |
| `bug`, `refactoring`, `bugfix`, `fix` |    🔧 Fixes & Refactoring     |
|       `build`, `ci`, `testing`        |    📦 Build System & CI/CD    |
|              `breaking`               |      💥 Breaking Changes      |
|            `documentation`            |       📝 Documentation        |
|            `dependencies`             |    ⬆️ Dependencies updates    |

## Checklist

- [ ] I have read the [`CODE_OF_CONDUCT.md`](https://github.com/featurebyte/featurebyte/blob/main/CODE_OF_CONDUCT.md) and [`CONTRIBUTING.md`](https://github.com/featurebyte/featurebyte/blob/main/CONTRIBUTING.md) guides.
- [ ] I have written tests for the changes made.
- [ ] I have written docstrings in [NumpyDoc format](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard)
- [ ] I have labeled my Pull Request correctly

### CODE REVIEW

**Review by jevonyeoh — APPROVED** [review]
Would we want to add this to docs as well?

**Comment by gee-senbong** [comment]
> Would we want to add this to docs as well?

Maybe we should. Will update this PR & also add the column cleaning info for the table columns & view columns (https://featurebyte.atlassian.net/browse/DEV-1703).

**Review by kchua78 — APPROVED** [review]

**Comment by gee-senbong** [comment]
Will merge this PR first & create another PR for DEV-1703

File: featurebyte/common/documentation/documentation_layout.py [hunk 1]
```diff
@@ -499,6 +499,7 @@ def _get_view_layout() -> List[DocLayoutItem]:
         DocLayoutItem([VIEW, INFO, "SCDView.surrogate_key_column"]),
         DocLayoutItem([VIEW, INFO, "View.columns"]),
         DocLayoutItem([VIEW, INFO, "View.columns_info"]),
+        DocLayoutItem([VIEW, INFO, "View.column_cleaning_operations"]),
```
Comment (gee-senbong) [inline featurebyte/common/documentation/documentation_layout.py:502]: @jevonyeoh Updated `_get_view_layout()` to include this.

**Pull request merged**